requirements_dev.txt

```
@@ -1,4 +1,4 @@
-nodejs
+nodejs >16,<17
```

Let's be more readable :wink: 

```suggestion
nodejs >=16,<17
```